src/test/ui/try-block/try-block-bad-type.stderr

```diff
@@ -32,18 +32,18 @@ LL |     let res: Result<i32, i32> = try { };
               found type `()`
 
 error[E0277]: the trait bound `(): std::ops::Try` is not satisfied
-  --> $DIR/try-block-bad-type.rs:17:23
+  --> $DIR/try-block-bad-type.rs:17:25
    |
 LL |     let res: () = try { };
-   |                       ^^^ the trait `std::ops::Try` is not implemented for `()`
+   |                         ^ the trait `std::ops::Try` is not implemented for `()`
```

Comment:
This change in spans seems unfortunate.

Reply:
Ok, I'll bring the old spans back, just thought it weird that the generated `from_ok` method got the span of the whole block. Of course the best span here is the one containing the type name :)